README.md

```diff
@@ -202,6 +202,7 @@ Evasions are the modules that help you to evade AV/EDR:
 | selfdelete     | Delete the current binary during runtime | None | |
 | sleep          | Sleep during a fixed amount of time in seconds. | Delay: the amount of time to sleep, default is 5s | |
 | createmutex    | Create a mutex with a specific name. | MutexName: the name of the mutex, default is "UruMutex" | |
+| refreshdll     | Refresh the given dll to remove hook by using the dll on disk. (Inspired by sliver/scarecrow and TimWhitez works). | UseBanana: UseBananaPhone to perform syscall. Default is "false", DllName: Name of the dll to refresh. Default is "C:\\\\Windows\\\\System32\\\\kernel32.dll". | Only work if windows version is "10.0" |
```

Comment:
Not the good line in the file, but the README need to change to list the supported languages.

We also need to add supported artifacts by languages, maybe:

|      Name      | Description | Argument(s) | Comment | C | Go |
|:--------------:|-------------|-------------|-----------|---|----|
| english-words  | Add a random number of english words to the binary. | NumberOfWord: define the number of english words to add to the binary between 1 and 1000. | Note that the words will no be obfuscated when using garble | ❎ |  ✔️  |

But comment might not correspond to the language..

Or something like this:

### Evasions
#### golang
here insert array with golang evasions
#### C  
here insert array with c evasions

Reply:
The first example with English words is super clean, and concise. Language specific comments (like what you provided) reads fine, might be worth noting the language the comment pertains to (for those who may not know)